### ISSUE TITLE
sailfish: settings.ini gets overwritten

### ISSUE THREAD

**Issue by t4k1t** [nemomobile/fingerterm#19]
Appearently ~/.config/FingerTerm/settings.ini gets overwritten every time you launch FingerTerm in Sailfish OS.

You can reproduce by e.g. setting visualBell to false and then relaunching FingerTerm - It will reset to true.

**Comment by oyvindaabling** [comment]
Yes, manually editing settings.ini (change visualBell to false) creates two identical %General sections, both with visualBell=true, on FingerTerm startup.
Could we *please* have something done about this - it's *really* annoying ?
Btw, if the About menu is displayed, the visualBell is turned Off (until next time), and the duplicate %General section goes away, but only after pressing OK.
See also https://together.jolla.com/question/14087/fingerterm-display-flashing/
